docs/user/index.md

```diff
@@ -102,20 +91,6 @@ Actions are set by means of the `action` field of the rule. Let's see an example
    }
 ```
 
-The `type` field is mandatory and must be one of the following:
-
--   `update` - creating or updating entities and attributes of those entities in the context broker.
-    [(update action details)](../API/plain_rules.md#update-attribute-action)
--   `sms` - sending a SMS. [(sms action details)](../API/plain_rules.md#sms-action)
--   `email` - sending an email. [(email action details)](../API/plain_rules.md#email-action)
--   `post` - making a HTTP request to a provided URL [(post action details)](../API/plain_rules.md#http-request-action)
--   `twitter` - sending a tweet [(twitter action details)](../API/plain_rules.md#twitter-action)
-
-An action can _optionally_ have a field called `interval`, aimed at limiting the frequency of the action execution (for
-the rule and entity which fired it). The value is expressed in milliseconds, and represents the minimum lapse between
-executions. Once the action is _successfully_ executed, it won't be executed again until that period of time has
-elapsed. All execution requests in between will be silently discarded.
-
```

Comment:
Removed due to redundancy? Where this information is now provided?

Reply:
As described in the next line
> You will find more information about Perseo Actions [here](../API/plain_rules.md#actions).

The part removed is a subset of plain_rules.md#actions

Reply:
NTC